app/views/components/pickadate.html

```diff
@@ -4,3 +4,8 @@
   </span>
   <input type="text" placeholder="Select a date..." class="form-control" ng-disabled="disabled">
 </div>
+<div ng-if="(minDate || maxDate) && (minDate != '' || maxDate != '')" class="help-block row" translate>
+   <label class="valid-range-msg">Valid Range:</label>
+   <span class="margin-left-10" ng-if="minDate && minDate!=''">Min Date- {{minDate | dateRangeFormat}} </span>   
+   <span class="margin-left-10" ng-if="maxDate && maxDate !=''"> Max Date- {{maxDate | dateRangeFormat}}</span>   
```

Comment:
I don't think you need the ` && maxDate !=''` here and a couple other places. An empty string should evaluate to false.